src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/ArmBase.PlatformNotSupported.cs

```diff
@@ -42,6 +42,16 @@ public abstract class Arm64
             ///   A64: CLZ Xd, Xn
             /// </summary>
             public static int LeadingZeroCount(ulong value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            ///   A64: RBIT Xd, Xn
+            /// </summary>
+            public static int ReverseElementBits(long  value) { throw new PlatformNotSupportedException(); }
```

Comment:
Can you remove redundant space between `long` and `value` and in other places as well?